platforms/nuttx/src/px4/stm/stm32_common/include/px4_arch/hw_description.h

```diff
@@ -91,6 +91,9 @@ enum Timer {
 	Timer12,
 	Timer13,
 	Timer14,
+#ifdef STM32_TIM15_BASE
```

Comment:
I think this is not needed here.

Reply:
I added this, because Timer15 does not exist on the STM32F4, which has its own specialized `hw_description.h` and will fail the build due to an unhandled switch statement if this ifdef is not included:

```
/opt/workspace/px4/platforms/nuttx/src/px4/stm/stm32f4/include/px4_arch/hw_description.h:42:9: error: enumeration value 'Timer15' not handled in switch [-Werror=switch]
   42 |  switch (timer) {
      |         ^
```

Reply:
OK got it.